src/Microsoft.AspNetCore.Razor.Language/AllowedChildTagDescriptor.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    [DebuggerDisplay("{DisplayName,nq}")]
+    [DebuggerDisplay("{" + nameof(DisplayName) + ",nq}")]
```

Comment:
After some discussion about this, we should make the following changes:
1. Override ToString where we have a display name property
2. Make a real `DebuggerToString()` for the places that don't use a display name property